### PR TITLE
Remove usage of deprecated Gradle features.

### LINKED ISSUE #60: Get rid of deprecated Gradle features
Hi, 
ktlint version "3.0.1"
gradle 4.5.1

STR:
run any default check (e.g. :ktlintMainCheck)
Actual result: warning appears:
```
Deprecated Gradle features were used in this build, making it incompatible with Gradle 5.0.
See https://docs.gradle.org/4.5.1/userguide/command_line_interface.html#sec:command_line_warnings
```
Expected result: no warning

### PR DESCRIPTION
**Pull request by Tapchicoma**
Fixes #60.

### CODE REVIEW

File: samples/android-app/build.gradle.kts [hunk 1]
```diff
@@ -10,19 +10,26 @@ apply {
...
         getByName("release") {
             isMinifyEnabled = false
             proguardFiles(getDefaultProguardFile("proguard-android.txt"), "proguard-rules.pro")
         }
     }
+
+    compileOptions {
+        setSourceCompatibility(JavaVersion.VERSION_1_8)
+        setTargetCompatibility(JavaVersion.VERSION_1_8)
+    }
```
Comment (eskatos) [inline samples/android-app/build.gradle.kts]: :x: Should the changes in this file be part of this PR?

Comment (JLLeitschuh) [reply samples/android-app/build.gradle.kts]: @Tapchicoma Can you explain why this was necessary. I'm fine with tossing in things that are unrelated as long as they are different commits and you note that the PR shouldn't be squashed.

Comment (Tapchicoma) [reply samples/android-app/build.gradle.kts]: It is not actually related to plugin, but related to deprecated warning when running plugin for samples. The warning was:

Comment (JLLeitschuh) [reply samples/android-app/build.gradle.kts]: Fair enough.

**Pull request merged**